service/src/test/java/bio/terra/pipelines/service/PipelinesServiceTest.java

```diff
@@ -227,6 +228,13 @@ void updatePipelineWorkspace() {
     // assert the fetched info from Rawls has been written
     assertEquals(newWorkspaceStorageContainerName, p.getWorkspaceStorageContainerName());
     assertEquals(newWorkspaceGoogleProject, p.getWorkspaceGoogleProject());
+
+    // update pipeline with warp-like wdlMethodVersion
+    pipelinesService.updatePipelineWorkspace(
```

Comment:
should we assert anything after this? i get that we just expect this to work without throwing an error, but it could theoretically silently fail somehow?

Reply:
im not sure how it could silently fail here.  do you mean like the repository doesn't save anything even though it says it did?